core/app_modules/user.js

```diff
@@ -51,5 +53,79 @@ module.exports = {
 
     assert(recordsAffected, 'Could not update information for ' + netid);
     return yield Promise.resolve();
+  }),
+
+  get: co.wrap(function*(netid) {
+    var client = db();
+    var query = "SELECT * FROM users WHERE netid = $1";
+    return yield Promise.resolve(yield client.query(query, [netid]));
+  }),
+
+  // accepts any number of NetIDs as arguments
+  ldapInfo: co.wrap(function*(netid) {
+    var data = {};
+
+    if(!arguments.length) {
+      throw new Error("No arguments passed.");
+    }
+
+    for(let i = 0; i<arguments.length; i++) {
+      if(arguments[i].match(/[^a-z0-9]/i)) {
+        throw new Error("Invalid NetID: " + arguments[i])
```

Comment:
It looks good, but I feel like I haven't adequately reviewed if it I don't make a comment, so I'm nitpicking. You missed a semicolon here.